third_party_licenses/dev_detailed.json

```diff
@@ -13,13 +13,19 @@
         "repository": "https://github.com/evanw/esbuild"
     },
     "@express-rate-limit/prettier@1.1.0": {
-        "licenses": "Custom: https://prettier.io/",
+        "licenses": "UNKNOWN",
         "repository": "https://github.com/express-rate-limit/prettier"
     },
     "@express-rate-limit/tsconfig@1.0.0": {
         "licenses": "UNKNOWN",
         "repository": "https://github.com/express-rate-limit/tsconfig"
```

Comment:
Oh, we should put licenses on these...

Reply:
Our tsconfig 1.0.2 and prettier 1.1.1 now have MIT licenses, if you'd like to update them in this PR

Reply:
Done